frontend/assets/translations/es.json

```diff
@@ -0,0 +1,921 @@
+        "submit":"Submit",
+        "select":"Select",
+        "environmentVar":"Environment Variables",
+        "saving":"Saving...",
+        "saveDatasource":"Save data source",
+        "authorize":"Authorize",
+        "connect":"Connect",
+        "reconnect":"Reconnect",
+        "components":"components",
+        "send":"Send",
+        "noConnection":"could not connect",
+        "connectionVerifeid":"connection verified",
+        "left":"Left",
+        "center":"Center",
+        "right":"Right",
+        "justified":"Justified",
+        "host":"Host",
+        "operation":"Operation",
+        "header":"HEADER",
+        "path":"PATH",
+        "query":"QUERY",
+        "requestBody":"REQUEST BODY"
+    },
+    "errorBoundary":"Something went wrong.",
+    "viewer":"Sorry!. This app is under maintenance",
+    "app":{
+        "updateAvailable":"Update available",
+        "newVersionReleased":"A new version of ToolJet has been released.",
+        "readReleaseNotes":"Read release notes & update",
+        "skipVersion":"Skip this version"
+    },
+    "stripe":"Please wait whle we load the OpenAPI specification for Stripe.",
+    "openApi":{
+        "noValidOpenApi":"Valid OpenAPI Spec is not available!.",
+        "selectHost":"Select a host",
+        "selectOperation":"Select an operation"
+    },
+    "slack":{
+        "authorize":"Authorize",
+        "connectToolJetToSlack":"ToolJet can connect to Slack and list users, send messages, etc. Please select appropriate permission scopes.",
+        "chatWrite":"chat:write",
+        "listUsersAndSendMessage":"Your ToolJet app will be able to list users and send messages to users & channels.",
+        "connectSlack":"Connect to Slack"
+    },
+    "googleSheets":{
+        "readOnly":"Read only",
+        "enableReadAndWrite":"If you want your ToolJet apps to modify your Google sheets, make sure to select read and write access",
+        "readDataFromSheets":"Your ToolJet apps can only read data from Google sheets",
+        "readWrite":"Read and write",
+        "readModifySheets":"Your ToolJet apps can read data from sheets, modify sheets, and more.",
+        "toGoogleSheets":"to Google Sheets"
+
+    },
+    "profile": {
+      "profileSettings": "Profile Settings"
+    },
+    "loginSignupPage":{
+        "forgotPassword":"Olvidaste la contraseña",
+        "emailAddress":"Dirección de correo electrónico",
+        "enterEmail":"Ingrese correo electrónico",
+        "resetPassword":"Restaurar contraseña",
+        "dontHaveAccount":"¿Aún no tienes una cuenta?",
+        "signIn":"Acceder",
+        "signUp":"Registrarte",
+        "createToolJetAccount":"Crear una cuenta de ToolJet",
+        "enterBusinessEmail":"Ingrese el correo electrónico de su empresa",
+        "alreadyHaveAnAccount":"¿Ya tiene una cuenta?",
+        "password":"Contraseña",
+        "showPassword":"Mostrar contraseña",
+        "loginTo":"Acceder a",
+        "yourAccount":"su cuenta",
```

Comment:
Hi, just a small detail here. To keep the consistency between the values I would recommend to replace the first character for a capital one `Su cuenta`.

The rest of the translations are great!

Reply:
I don't know how I missed it. Thanks for pointing it out.